x/ccv/provider/keeper/proposal.go

```diff
@@ -255,6 +255,12 @@ func (k Keeper) MakeConsumerGenesis(
 		return false
 	})
 
+	// currentValidators := k.GetOptedIn(ctx, chainID)
```

Comment:
Leaving those commented for now to show how `ComputeValidatorUpdates`, etc. are going to be used.